docker/preload_models.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import pooch
+
+def load_models(version):
```

Comment:
This code is a duplicate from cli.py, and instead should use the [load_models](https://github.com/icatcherplus/icatcher_plus/blob/30c0199f9218e99a3589067aad6e24fe68bf583b/src/icatcher/cli.py#L155) function from the cli directly so it can be maintained properly.
If needed, the load_models function can be extended with arguments to suit the needs of the docker ("download_only" or something like this).